Formula/zsh-syntax-highlighting.rb

```diff
@@ -27,7 +27,7 @@ def caveats
 
       source ~/.zshrc
 
-    Additionally, if you receive "highlighters directory not found" error message,
+    Additionally, if you a receive "highlighters directory not found" error message,
```

Comment:
This looks like a typo, `Additionally, if you a receive "highlighters directory not found" error message` => `Additionally, if you receive a "highlighters directory not found" error message`, but actually I think that we are good with what is there now.